src/core/templates/buttons.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="buttons-display">
         <div class="buttons-marking-icon icon display-none">
         </div>
-        <div class="buttons-display-inner">
+        <div class="buttons-display-inner" aria-live="rude">
```

Comment:
Ok @oliverfoster so this is a bit strange. It doesn't seem to behave itself on Mac osx + safari + voiceover, I was able to get it working intermittently but not consistently. However, on chrome + voiceover it works perfectly without any issues. It's odd because it also seems to work consistently on safari on iOS + iOS voiceover. 

I think this suggests that there could be a bug in Safari because I did hear it several times, intermittently.